lib/ui/views/grade_details_view.dart

```diff
@@ -0,0 +1,300 @@
+                              model.course.summary.median),
+                        ),
+                      ),
+                    ),
+                    Expanded(
+                      flex: 3,
+                      child: getHeadersSummary(
+                        AppIntl.of(context).grades_standard_deviation,
+                        validateGrade(
+                          context,
+                          model.course.summary.standardDeviation.toString(),
+                          model.course.summary.standardDeviation.toString(),
+                        ),
+                      ),
+                    ),
+                    Expanded(
+                      flex: 3,
+                      child: getHeadersSummary(
+                        AppIntl.of(context).grades_percentile_rank,
+                        validateGrade(
+                          context,
+                          model.course.summary.percentileRank.toString(),
+                          model.course.summary.percentileRank.toString(),
+                        ),
+                      ),
+                    ),
+                  ]),
+              Column(children: <Widget>[
+                for (var evaluation in model.course.summary.evaluations)
+                  GradeEvaluationTile(evaluation,
+                      key: Key("GradeEvaluationTile_${evaluation.title}")),
+              ]),
+            ],
+          ),
+        ],
+      );
+    } else if (model.isBusy) {
+      return const Center(child: CircularProgressIndicator());
+    } else {
+      return Center(
+        child: GradeNotAvailable(
+            key: const Key("GradeNotAvailable"), onPressed: model.refresh),
+      );
+    }
+  }
+
+  Align _buildClassInfo(String info) =>
+    Align(
+      alignment: Alignment.centerLeft,
+      child: Padding(
+        padding: const EdgeInsets.only(left: 15.0),
+        child: Text(
+          info,
+          style: const TextStyle(color: Colors.white, fontSize: 16),
+          overflow: TextOverflow.ellipsis,
+        ),
+      ),
+    );
+
+  Column _buildHeadersSummary(
+      double grade, String recipient, Color color, BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        FittedBox(
+            fit: BoxFit.fitWidth,
+            child: Text(
+              AppIntl.of(context).grades_grade_with_percentage(grade, 100, grade),
+              style:
+                  Theme.of(context).textTheme.headline6.copyWith(color: color)),
+        ),
+        Text(recipient,
+            style: Theme.of(context).textTheme.bodyText1.copyWith(color: color)
+            ),
+      ],
+    );
+  }
+
+  String validateGrade(BuildContext context, String grade, String text) {
+    if (grade == "null" || grade == null) {
+      return AppIntl.of(context).grades_not_available;
+    }
+
+    return text;
+  }
+
+  SizedBox getHeadersSummary(String title, String number) {
```

Comment:
```suggestion
  SizedBox _buildHeadersSummary(String title, String number) {
```

Can you also add a little doc to explain what is inside the header summary? :smile: